statistics.py

```diff
@@ -163,13 +164,10 @@ def print_most_often_replies_to(chat: Chat) -> None:
 
 
 def _parse_string_to_words(text: str) -> List[str]:
-    word_tuples = re.findall("([a-zA-Zа-яА-Я]+-[a-zA-Zа-яА-Я]+)|([a-zA-Zа-яА-Я]+)", text)
+    word_tuples = regex.findall(WORD_REGEX, text)
     words = []
-    for (first_match, second_match) in word_tuples:
-        if first_match != "":
-            words.append(first_match)
-        else:
-            words.append(second_match)
+    for (match, first_match, second_match) in word_tuples:
+        words.append(match)
```

Comment:
```suggestion
    for match in word_tuples:
        words.append(match[0])
```

Reply:
Или вот так. Но я не уверен, что в питоне так делают, и что какой-нибудь потенциальный mypy не будет ругаться.
```suggestion
    for (match, _, _) in word_tuples:
        words.append(match)
```

Reply:
Иначе у тебя получаются неиспользованные переменные, что не очень хорошо